src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -116,6 +121,18 @@ public MafOutputRenderer(final Path outputFilePath,
         this.inputFileHeader = inputFileHeader;
         dataSourceFactories = dataSources;
 
+        this.tnPairs = SamplePairExtractor.extractPossibleTumorNormalPairs(this.inputFileHeader);
```

Comment:
Can you add in a new comment in the constructor and the output format argument for MAF saying that this should not be used for Germline data?

Reply:
Can I just say it is unsupported?  People are welcome to use it -- might be useful with some big caveats....

Done.